kivy/lib/osc/OSC.py

```diff
@@ -177,24 +231,45 @@ def OSCArgument(data):
     """Convert some Python types to their
     OSC binary representations, returning a
     (typetag, data) tuple."""
+        
+    try:
```

Comment:
Any particular reason to eat exceptions here? Did you have a specific issue to fix?